src/apps/components/AppUrlLink/AppUrlLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { LinkIcon, Pill } from "@saleor/macaw-ui";
+import React from "react";
+
+interface AppUrlLinkProps {
+  url: string | null;
+  onClick: () => void;
+}
+
+const AppUrlLink: React.FC<AppUrlLinkProps> = ({ url, onClick }) => {
+  if (typeof url === "string") {
```

Comment:
Why not just the following?
```suggestion
  if (url) {
```